BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -2479,7 +2479,7 @@ void SLWGuidance()
                 DoAero(SLWTarget);
             }
 
-            if (SLWTarget.y > 0f) SLWTarget.y = getSWLWOffset;
+            if (SLWTarget.y > 0f) SLWTarget.y = getSWLWOffset; //shouldn't this be *before* the DoAero() to send the torp towards the target position?
```

Comment:
Yep. `SLWTarget` is going out of scope after this, so this line is pointless if it's not before `DoAero`.